### PR TITLE
Organised and Added Components

### PR DESCRIPTION
**Pull request by Tien9320**
Organised components and stylesheet's folders

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 // Your CSS partials
 @import "components/index";
+@import "components/banner";
```
Comment (MmeIsabelle) [inline app/assets/stylesheets/application.scss:13]: Don't import those 2 here... Add them to the _index.scss file.

File: app/assets/stylesheets/components/_banner.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.banner {
+  background-size: cover;
+  background-position: center;
+  padding: 150px 0;
+}
+
+.banner h1 {
```
Comment (MmeIsabelle) [inline app/assets/stylesheets/components/_banner.scss:7]: You want to use nesting instead? Try to put your h1 and p inside the .banner

File: app/assets/stylesheets/components/_searchform.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
+.search-form-control {
+  position: relative;
+}
+
+.search-form-control .btn {
```
Comment (MmeIsabelle) [inline app/assets/stylesheets/components/_searchform.scss:5]: Same comment here. Nesting would work.

**Review by MmeIsabelle — COMMENTED** [review]
@Tien9320 see my comments inside the code

File: app/assets/stylesheets/components/_searchform.scss (hunk 3 above)
Comment (Tien9320) [reply app/assets/stylesheets/components/_searchform.scss]: omg thank you, I start forgetting which components was created

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (MmeIsabelle) [reply app/assets/stylesheets/application.scss:13]: Your _banner and _searchform should be included in this list
image

File: app/assets/stylesheets/components/_banner.scss [hunk 4]
```diff
@@ -0,0 +1,23 @@
+.banner {
+  background-size: cover;
+  background-position: center;
+  padding: 150px 0;
+}
+
+.banner {
+  h1{
```
Comment (MmeIsabelle) [inline app/assets/stylesheets/components/_banner.scss:8]: Just bring line 8 to 14 after line 4

File: app/assets/stylesheets/components/_banner.scss [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    margin: 0;
+  color: white;
+  text-shadow: 1px 1px 3px rgba(0,0,0,0.2);
+  font-size: 32px;
+  font-weight: bold;
+  }
+
+  p {
```
Comment (MmeIsabelle) [inline app/assets/stylesheets/components/_banner.scss:16]: Move line 6 to 21 after the h1 inside the banner

File: app/assets/stylesheets/components/_searchform.scss [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+  position: relative;
+}
+
+.search-form-control {
+  .btn {
+  position: absolute;
+  top: 8px;
+  bottom: 8px;
+  right: 8px;
+}
```
Comment (MmeIsabelle) [inline app/assets/stylesheets/components/_searchform.scss:11]: Missing one }

Comment (MmeIsabelle) [inline app/assets/stylesheets/components/_searchform.scss:11]: You are missing a }, unless the following lines are all nested. If it is the case, indent them.

Comment (MmeIsabelle) [inline app/assets/stylesheets/components/_searchform.scss:11]: Line 11 to 21 need to be indented

**Comment by MmeIsabelle** [comment]
This looks great now!

**Pull request merged**